src/main/scala/edu/ie3/simona/config/ArgsParser.scala

```diff
@@ -225,8 +225,8 @@ object ArgsParser extends LazyLogging {
       ConfigFactory.parseString(
         s"""config = "${parsedArgs.configLocation.get.replace("\\", "\\\\")}"
            |simona.runtime_configuration {
```

Comment:
Also this is supposed to be `simona.runtime`?

Reply:
I don't know. I checked this part and it was `simona.runtime_configuration` in the initial commit.

Reply:
Do you know, if we actually need this part? It is given to the `ActorSystem`, but I don't see any place it is ever used.

Reply:
I think we can just throw out `selectedSubnets` and `selectedVoltLvls` from `ArgsParser`, now that I think about it. I think we never really used it, and if one wants to use it, it can still be configured through the conf file

Reply:
But then also, there's lots of stuff in ArgsParser that can be thrown out. So maybe we should create a separate issue for that.

Reply:
#1192 -> This PR can just stay as it is in that regard